tests/test_mercurial.py

```diff
@@ -203,7 +205,9 @@ def _readme(content):
     assert tip.node != base
     assert (
         tip.desc
-        == b"try_task_config for code-review\nDifferential Diff: PHID-DIFF-solo"
+        == b"""try_task_config for code-review
```

Comment:
I think the phab request should be first in the list

Reply:
Done